web/src/components/VisibleChanges.res

```diff
@@ -0,0 +1,24 @@
+let use = () => {
+  let (isChecked, setToggle) = React.useState(_ => false)
+  let label = switch isChecked {
+  | true => "Hide changes"
```

Comment:
I think we should improve the labeling thing.
| true => "Hide hidden/masked changes"
| false => "Reveal hidden/masked changes"